### DISCUSSION TITLE
Anyone knows, how to stop turining cool arrow-shaped segment of status bar into rectangles?

### DISCUSSION THREAD

**Discussion by voodookiidoo** [erikw/tmux-powerline#342] [Q&A]
If i have an active proccess, such as vim or cat, the tab in the status bar that respresents the window will be rectengular shaped, instead of arrow-like. How do I fix that?

**Comment by erikw** [comment]
Hey, I’m not quite sure what you mean. Could you include a screenshot with an arrow?

**Reply by voodookiidoo** [reply]
okay so apparently there is this statusbar
<img width="995" alt="image" src="https://github.com/erikw/tmux-powerline/assets/65444512/748625ac-aac5-4bab-b7b5-8f7ff958cfd7">
Everything is fine. But when there is an active process in an unactive window (spotify-tui, vim) or for example sleep process finishes, it turns into
<img width="936" alt="image" src="https://github.com/erikw/tmux-powerline/assets/65444512/d9c29ddb-0fa8-4266-9753-b54f80725168">
<img width="595" alt="image" src="https://github.com/erikw/tmux-powerline/assets/65444512/66cababa-f196-44ef-a374-1c46e1a5af3d">


What could possibly be the case?
Tried diving into configs but i'm not that smart at configuring something, so i could really use some help from someone experienced. Please, help me :(

**Reply by erikw** [reply]
Oh interesting. I’m not sure. Though it should be possible to figure out with 
https://github.com/erikw/tmux-powerline#debugging

Time-wise I’m probably not able to dig in to this right now.

**Reply by voodookiidoo** [reply]
i guess this happens because tmux counts more than one window as active. is there an option to limit it?

**Reply by embe221ed** [reply]
isn't this somehow related to the tmux bell? https://stackoverflow.com/a/39180062/7840511

**Reply by embe221ed** [reply]
@voodookiidoo does your 👍 reaction mean that it was the issue? If so, we should probably create a feature request to handle the bell and close the original issue.

**Comment by xx4h** [comment]
FTR, the screenshots supplied by @voodookiidoo show the effect of the default tmux setting for `window-status-bell-style` and `window-status-activity-style`, which are both `reverse`.

To fix this, we have basically two options

## Just set the default layout as bell style
```
tmux set-option -g window-status-bell-style default
tmux set-option -g window-status-activity-style default
# or set it empty
tmux set-option -g window-status-bell-style ""
tmux set-option -g window-status-activity-style ""
```  
This will just show the window as any other, but keep the window indicators (`!` for bell, `#` for activity).

## Create a theme based on default theme and add handling for the bell and activity
For this to work you will also have to set the default as mentioned in 1., on top of that,
as an example, change the following line: https://github.com/erikw/tmux-powerline/blob/c7b13f81ff86093b20923812557303c30575135d/themes/default.sh#L50
to
```
" #{?window_activity_flag,#[fg=green],}#{?window_bell_flag,#[fg=red],}#W "
 ```
 to get this for a window with bell (for the example above, keep in mind, that the later one has precedence, so if we have only a activity, the font will be green, if it has activity&bell, or only bell, it will be red):
 ![image](https://github.com/erikw/tmux-powerline/assets/2880059/0862cb5b-cee7-4f5e-9bb1-4e089b3561ce)
 If you forgot to set the `window-status-bell-style` from 1., it will look like this
 ![image](https://github.com/erikw/tmux-powerline/assets/2880059/f6edbbf7-2908-4291-84ee-243d064d6174)
